.github/workflows/ci.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
     runs-on: ${{ matrix.platform }}
     env:
       GOPATH: /home/runner/work/kpt-functions-sdk/go
-      GO111MODULE: off  
+      GO111MODULE: on
```

Comment:
There are other workflows that need to be updated as well (for releases)

Reply:
Thanks, updated releases. Removed setting go env variables for releasing ts pkg.